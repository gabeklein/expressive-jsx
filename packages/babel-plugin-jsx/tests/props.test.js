transform("will convert `./../file.ext` to require", `
  () => {
    <img src="./logo.svg">
      This should be a require-sourced file!
    </img>
  }
`);