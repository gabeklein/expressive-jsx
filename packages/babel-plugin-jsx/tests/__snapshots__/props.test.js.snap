// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`will convert \`./../file.ext\` to require 1`] = `
  () => (
    <img src={require("./logo.svg")}>
      This should be a require-sourced file!
    </img>
  );
`;
